index.js

```diff
@@ -1361,6 +1366,7 @@ instance.prototype.action = function (action, extras) {
 						self.log('error', 'Shell command failed. Guru meditation: ' + JSON.stringify(error))
 						self.debug(error)
 					}
+					self.system.emit('custom_variable_set_value', action.options.targetVariable, stdout)
```

Comment:
What if the user doesn't want to save the result into a variable? I dont know if the `internal:custom_variable` is able to have a 'none' option right now, but I think we need an option to make it include that
I suspect this will need a guard to check that `action.options.targetVariable` has a value to avoid it throwing a weird error

Reply:
I've replaced the `internal:custom_variable` with the map over all custom variable which should do the trick(?)

Reply:
Should I just add another action that includes saving to a variable?

Reply:
hold off on that, Ill have a quick play

Reply:
I've added a property to that special control `includeNone` which allows for this.
we dont have guards on the other calls to `self.system.emit('custom_variable_set_value',`, so I dont think we should bother doing so here